src/App.jsx

```diff
@@ -8,7 +9,9 @@ export const App = () => (
     <p>Sum of 10 and 0 is 10</p>
     <p>Sum of 0 and 5 is 5</p>
     <p>Sum of 0 and 0 is 0</p>
```

Comment:
remove this duplicate 
![image](https://user-images.githubusercontent.com/70009205/208151473-2246e0a4-f7fb-46b9-9635-2eb0ff3dbead.png)